bin/boilerplate/_config.yml

```diff
@@ -2,7 +2,7 @@
 # Values for this lesson.
 #------------------------------------------------------------
 
-# Which carpentry is this ("swc", "dc", or "lc")?
+# Which carpentry is this ("swc", "dc", "lc", or "cp")?
 carpentry: "swc"
```

Comment:
would it help to add a clarifying comment as to what `cp` and, perhaps, other abbreviations mean?